extensions/sql-bindings/src/test/dialog/addSqlBindingQuickpick.test.ts

```diff
@@ -65,13 +79,22 @@ describe('Add SQL Binding quick pick', () => {
 		const spy = sinon.spy(vscode.window, 'showErrorMessage');
 
 		// select Azure function
-		let quickpickStub = sinon.stub(vscode.window, 'showQuickPick').onFirstCall().resolves({ label: 'af1' });
+		let quickpickStub = sinon.stub(vscode.window, 'showQuickPick').returns(Promise.resolve('af1') as any);
 		// select input or output binding
 		quickpickStub.onSecondCall().resolves(<any>{ label: constants.input, type: BindingType.input });
-		// give object name
-		let inputBoxStub = sinon.stub(vscode.window, 'showInputBox').onFirstCall().resolves('dbo.table1');
+		sinon.stub(azureFunctionUtils, 'getAFProjectContainingFile').resolves(vscode.Uri.file('testUri'));
+		// select connection profile - create new
+		quickpickStub.onThirdCall().resolves(<any>{ label: constants.createNewLocalAppSettingWithIcon });
```

Comment:
It would be great to see more tests added for all the various paths users can take - right now we mostly rely on manual testing for that stuff which isn't great

Reply:
Absolutely this is will my plan post build release bugs fixes. I set this test up to ensure that it goes through the entire process properly and therefore will be much easier to add more tests later.